app/templates/home.html

```diff
@@ -1,5 +1,8 @@
+{% load socialaccount %}
```

Comment:
Where does this come from?

Reply:
those are template tags which are a part of django-allauth that is required for login to work